sda-download/biohackathon/paper.md

```diff
@@ -95,17 +95,17 @@ which enhances the security of the data access interfaces.
 
 # Results
 
-In order to achieve the project goals, both htsget-rs and sda-download need to be externted.
+In order to achieve the project goals, both htsget-rs and sda-download needed to be extened.
 The following sections describe the work done on each of the services.
 
 
 ## HTSGet
 
-In order to enable for random data access on encrypted files, we worked on
-extending htsget-rs [@htsget-rs] to work with the new version of the sda-download.
-We developed the following sequence diagram for interactions between the server
-softwares. Basically it means the client first communicates with the Rust server
-to get information on what to download, which it then downloads from the sda-download service.
+In order to enable for random data access on encrypted files, we
+extended htsget-rs [@htsget-rs] to work with the new version of the sda-download.
```

Comment:
which new version? the one that is described below?

Reply:
the version that it was developed during the biohackathon